form.js

```diff
@@ -0,0 +1,76 @@
+
+let form = document.forms[0];
+let userName = document.getElementById('name');
+let userSurname = document.getElementById('surname');
+let street = document.getElementById('street');
+let house = document.getElementById('house');
+let flat = document.getElementById('flat');
+
+ ///// Check data validity
+ function getToday() {
+    let now = new Date().toString();
+    let year = now.slice(11,15);
+    let day = +now.slice(8,10) + 1;
+    let monthStr = now.slice(4,7);
+    let monthList = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+    let month = monthList.indexOf(monthStr) + 1;
+    let minDay = `${year}-${month}-${day}`;
+    return minDay;
+}
+let dateOfDelivery = document.getElementById('date');
+    dateOfDelivery.setAttribute('min', getToday());
+
+///// Check gifts validity
+let gift = document.querySelector('select');
+    gift.addEventListener('click',(event) => {
+        let sum = 0;
+        for (const option of gift.options) {
+            if (option.selected) {
+                sum++;
+                if (sum > 2) {
+                    event.target.selected = false;
+                }
+            }
+        }
+    });
+
+///// Buttons and whole form validation
+function toValidate(form) {
+    complete.disabled = !form.checkValidity();
+}
+
+let error = document.querySelector('span');
```

Comment:
```suggestion
let error = document.querySelector('.error');
```
I can see two spans but only one of them is error. Of course, querySelector will return the first span, but better use class here.
![image](https://user-images.githubusercontent.com/78261193/209578268-a65661dd-eb97-4b9a-ae63-18933538a598.png)

Reply:
done👍 it`s logical